src/__tests__/userData.spec.ts

```diff
@@ -1,9 +1,92 @@
 import request from 'supertest';
 import app from '../app';
+import assert from 'assert';
 
 describe('User Data Tests', () => {
   it('should fetch a single user', async () => {
     const response = await request(app).get('/jambobjones');
     expect(response.body.username).toBe('jambobjones');
+    [
+      'username',
+      'name',
+      'birthday',
+      'avatar',
+      'ranking',
+      'reputation',
+      'gitHub',
+      'twitter',
+      'linkedIN',
+      'website',
+      'country',
+      'company',
+      'school',
+      'skillTags',
+      'about',
+    ].forEach((key) => {
+      assert(key in response.body);
+    });
+  });
```

Comment:
I followed this pattern of validation because my personal profile doesn't have much data associated with it, but at least you can guarantee to your users that they will receive a response object with these keys